aws_msk/dashboards/aws-msk.json

```diff
@@ -0,0 +1,313 @@
+{
```

Comment:
Each dashboard just needs to be under its own directory, instead of `aws_msk/dashboards/aws-msk.json` it would be `aws_msk/dashboards/aws-msk/aws-msk.json`.

Reply:
Hello @aswanson-nr Yes, would love to get this added to the existing quickstart. Please go ahead and merge it with the existing one . Reg the folder structure, will you go ahead and do that, please?

Reply:
Hey @ganeshmuralidhar! Sorry for the delay, I put up a new PR that moves this dashboard into the existing quickstart. Could you take a look and toss your approval on it? #1084